.github/pull_request_template.md

```diff
@@ -1,38 +1,38 @@
-# [Work Item ID or Title](./link-to-the-work-item)
+For more information about how to contribute to this repo, visit [this page](https://github.com/microsoft/FluidFramework/blob/main/CONTRIBUTING.md).
 
-For more information about how to contribute to this repo, visit this [page](https://github.com/microsoft/FluidFramework/blob/main/CONTRIBUTING.md)
+For specific guidelines for Pull Requests in this repo, visit [this wiki page](https://github.com/microsoft/FluidFramework/wiki/PR-Guidelines#guidelines).
 
-## Description
+The sections included below are suggestions for what you may want to include.
+Feel free to remove or alter parts of this template that do not offer value for your specific change.
 
-> Should include a concise description of the changes (bug or feature), it's impact, along with a summary of the solution
+## Description
 
-## Steps to Reproduce Bug and Validate Solution
+> A concise description of the changes (bug or feature) and their impact/motivation.
+> If this description is short enough to be used as the title, delete this section and just use the title.
 
-> Only applicable if the work is to address a bug. Please remove this section if the work is for a feature or story
-> Provide details on the environment the bug is found, and detailed steps to recreate the bug.
-> This should be detailed enough for a team member to confirm that the bug no longer occurs
+> For bug fixes, also include specifics of how to reproduce it / confirm it is fixed.
 
-## PR Checklist
+> If this Pull Request should close/resolve any issues when merged, use [the special syntax for that](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) here.
 
-> Use the check-list below to ensure your branch is ready for PR. If the item is not applicable, leave it blank.
+## Reviewer Guidance
 
--   [ ] I have updated the documentation accordingly.
--   [ ] I have added tests to cover my changes.
--   [ ] All new and existing tests passed.
--   [ ] My code follows the code style of this project.
--   [ ] I ran the lint checks which produced no new errors nor warnings for my changes.
--   [ ] I have checked to ensure there aren't other open Pull Requests for the same update/change.
+> List any specific things you want to get reviewer opinions on, and anything a reviewer would need to know to review this PR effectively.
+> Things you might want to include:
+>
+> -   Questions about how to properly make automated tests for your changes.
+> -   Questions about design choices you made.
+> -   Descriptions of how to manually test the changes (and how much of that you have done).
+> -   etc.
+>
+> If you have any questions in this section, consider making the PR a draft until all questions have been resolved.
 
 ## Does this introduce a breaking change?
 
 -   [ ] Yes
 -   [ ] No
 
```

Comment:
I'd also get rid of the checkboxes, the header plus the instructions seems enough. If anything, add "If no breaking change, remove this section".

Reply:
Sound good. I think that also fixes that github considers checkboxes "tasks" and lists you as having one unfinished if you don't check both boxes. I'll make the change.

Reply:
Done